src/Native/System.Native/pal_networking.cpp

```diff
@@ -41,7 +41,10 @@ enum
     HOST_ENTRY_HANDLE_HOSTENT = 2,
 };
 
-const int INET6_ADDRSTRLEN_MANAGED = 65; // The C# code has a longer max string length
+enum
+{
+    INET6_ADDRSTRLEN_MANAGED = 65 // Managed code has a longer max IPv6 string length
+};
```

Comment:
It avoids the need to silence the compiler warning, since it's no longer a variable. Just a minor thing.